Panels/PanelTable.js

```diff
@@ -803,14 +803,19 @@ define([
                     }
                 });
                 data += line + '\n';
+                const escapeMap = {'\n': '\\n', '\t': '\\t', '\r': '\\r', '\\': ''};
```

Comment:
`'\\'` should be expanded to `'\\\\'`. But I think this is a bit unsafe. 

In Python for `x='hello\nworld'`, one could do:`x.encode('unicode_escape').decode()`. 

I think you can achive the same with`JSON.stringify(x))` and then removing the first and last character (`"`). 
This would not be a good approach for generating a tsv file, but purely for showing stuff in the table.
Generating the tsv file should be done on the backend where there are nice tsv libraries

Reply:
Generating the tsv file on the backend means that we have to make a visualisation model library for each table resource that implements the `content2DisplayString` for all known models and fields.